### ISSUE TITLE
Workflow fails when a new issue already belongs to a project

### ISSUE THREAD

**Issue by jacobian** [takanabe/add-new-issues-to-project-column#19]
If the workflow runs on an issue that already exists in a project, it'll fail. 

This can happen when convverting a note on the board into an issue, i.e.:

1. On the project board automated with this action, create a new note (with the ➕ button on the column)
2. In the drop-down menu on the note, choose "Convert to issue"
3. Note that the workflow fails.

Here's the error:

```
Run docker://takanabe/github-actions-automate-projects:v0.0.1
  env:
    GITHUB_TOKEN: ***
    GITHUB_PROJECT_URL: [REDACTED]
    GITHUB_PROJECT_COLUMN_NAME: [REDACTED]
/usr/bin/docker run --name takanabegithubactionsautomateprojectsv001_3c592a --label e87b52 --workdir /github/workspace --rm -e GITHUB_TOKEN -e GITHUB_PROJECT_URL -e GITHUB_PROJECT_COLUMN_NAME -e HOME -e GITHUB_REF -e GITHUB_SHA -e GITHUB_REPOSITORY -e GITHUB_RUN_ID -e GITHUB_RUN_NUMBER -e GITHUB_ACTOR -e GITHUB_WORKFLOW -e GITHUB_HEAD_REF -e GITHUB_BASE_REF -e GITHUB_EVENT_NAME -e GITHUB_WORKSPACE -e GITHUB_ACTION -e GITHUB_EVENT_PATH -e RUNNER_OS -e RUNNER_TOOL_CACHE -e RUNNER_TEMP -e RUNNER_WORKSPACE -e ACTIONS_RUNTIME_URL -e ACTIONS_RUNTIME_TOKEN -e GITHUB_ACTIONS=true -v "/var/run/docker.sock":"/var/run/docker.sock" -v "/home/runner/work/_temp/_github_home":"/github/home" -v "/home/runner/work/_temp/_github_workflow":"/github/workflow" -v "/home/runner/work/outcome-origination-platform/outcome-origination-platform":"/github/workspace" takanabe/github-actions-automate-projects:v0.0.1
2020/02/21 17:00:39 [INFO] Event name: issues
2020/02/21 17:00:39 [INFO] Payload for issues extract correctly
2020/02/21 17:00:39 [INFO] project url: [REDACTED]
2020/02/21 17:00:39 [INFO] Project Name: [REDACTED]
2020/02/21 17:00:39 [INFO] Project type:repository
2020/02/21 17:00:39 [INFO] Column Name: [REDACTED]
2020/02/21 17:00:39 [INFO] Project card is being added to column [REDACTED]
2020/02/21 17:00:40 [ERROR] POST https://api.github.com/projects/columns/[REDACTED]/cards: 422 Validation Failed [{Resource:ProjectCard Field:data Code:unprocessable Message:Project already has the associated issue}]
Failed to get results from GitHub
main.validateGitHubResponse
	/app/main.go:264
main.addToProject
	/app/main.go:239
main.main
	/app/main.go:86
runtime.main
=```

**Comment by knumat** [comment]
One workaround for this issue is to add `continue-on-error: true` to the github-actions-automate-projects step(s) of the job in your workflow .yml file.  This allows the job/workflow to succeed even if the project assignment step fails (i.e. because it was already assigned), eliminating the annoying failure emails.  More info:
https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstepscontinue-on-error

**Comment by takanabe** [comment]
this will be solved with the latest docker image or v0.0.3 which I'm preparing 

Related PR: https://github.com/takanabe/github-actions-automate-projects/pull/26
